spec.md

```diff
@@ -160,7 +161,7 @@ If the digest does differ, it MAY be the case that the hashing algorithms used d
 See [Content Digests](https://github.com/opencontainers/image-spec/blob/v1.0.1/descriptor.md#digests) <sup>[apdx-3](#appendix)</sup> for information on how to detect the hashing algorithm in use.
 Most clients MAY ignore the value, but if it is used, the client MUST verify the value against the uploaded blob data.
 
-If the manifest is not found in the registry, the response code MUST be `404 Not Found`.
+If the manifest is not found in the repository, the response code MUST be `404 Not Found`.
```

Comment:
repository is optional here...

Reply:
All of these "in the registry" statements could almost just be removed. I agree it is up to the implementation to determine the scope of a blob or manifest. Repository scope makes the most sense but that might need more explanation above or as part of the implementation guide.